twine-upload.sh

```diff
@@ -43,7 +43,7 @@ INPUT_PRINT_HASH="$(get-normalized-input 'print-hash')"
 if [[ "${INPUT_USER}" == "__token__" && -z "${INPUT_PASSWORD}" ]] ; then
     # No password supplied by the user implies that we're in the OIDC flow;
     # retrieve the OIDC credential and exchange it for a PyPI API token.
-    echo "::notice::In OIDC flow"
+    echo "::notice::Attempting to perform OIDC credential exchange"
```

Comment:
@woodruffw would you mind extending this with the suggestion outlined @ https://github.com/pypi/pypi-oidc-private-beta-community/issues/15#issuecomment-1479696511 to cover all the branches and include reasonable details?

Reply:
Yeah, can do now!

Reply:
It'd be good to align this message with the ones below too.
Maybe smth like
```suggestion
    echo \
        '::notice::Attempting to perform OIDC credential exchange ' \
        'to retrieve a temporary short-lived API token for authentication ' \
        "against ${INPUT_REPOSITORY_URL}"
```